azurelinuxagent/ga/policy/policy_engine.py

```diff
@@ -130,7 +129,7 @@ def evaluate_query(self, input_to_check, query):
             "extensions": {
                 "<extension_name_1>": {
                     "signingInfo": {
-                        "extensionSigned": <true, false>
+                        "extensionSigned": <True, False>
```

Comment:
Nit: in Json, Boolean value is represented in lower case.

Reply:
Expected format is a python dict. Booleans are capitalized in Python but converted to lowercase with the json.dump() call in eval_query().